### PR TITLE
improvement: disable `anchor` button on pending anchor

### LINKED ISSUE #2079: Disable `Anchor project` button if there is a pending anchor
If multiple anchors are pending, only one will be executable. This means that if two people are anchoring a project, the first one to execute will remove the other one that was pending. To avoid confusion, we could disable the `anchor project` button when there is already a pending anchor.

### PR DESCRIPTION
**Pull request by juliendonck**
closes #2079 
Signed-off-by: Julien Donck <hello@juliendonck.com>

### CODE REVIEW

File: ui/Screen/Org.svelte [hunk 1]
```diff
@@ -89,6 +89,14 @@
       },
     ];
   };
+  console.log(activeTab);
```
Comment (rudolfs) [inline ui/Screen/Org.svelte:92]: 👀

File: ui/Screen/Org.svelte [hunk 2]
```diff
@@ -113,7 +121,8 @@
         <ProjectsMenu
           orgAddress={address}
           {gnosisSafeAddress}
-          disabled={activeTab.projectCount === 0} />
+          disabled={activeTab.projectCount === 0 || pendingAnchor}
```
Comment (rudolfs) [inline ui/Screen/Org.svelte:123]: Now that the logic to disable this button is a bit more complex we should just pass in project count to the `ProjectsMenu` component as a prop and do all the logic inside the component.

File: ui/Screen/Org.svelte [hunk 3]
```diff
@@ -89,6 +89,13 @@
       },
     ];
   };
+
+  let pendingAnchor: boolean = false;
+  $: {
+    pendingAnchor = activeTab.anchoredProjects.some(
+      proj => proj.anchor.type === "pending"
```
Comment (rudolfs) [inline ui/Screen/Org.svelte:96]: We should match against the type, not an arbitrary string here.

```suggestion
      proj => proj.anchor.type === project.PendingAnchor
```

And import the type from project at the top:
```
import type * as project from "ui/src/project"
```

File: ui/Screen/Org.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/Screen/Org.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/2085/commits/8f523c9e166bcdf4b4f99eb94b50bb0580ec17c7

File: ui/Screen/Org.svelte (hunk 3 above)
Comment (rudolfs) [reply ui/Screen/Org.svelte]: Yea, disregard this. It doesn't make sense.

**Comment by rudolfs** [comment]
Pushed a fix that covers pending anchors in resolved projects as well as unresolved: https://github.com/radicle-dev/radicle-upstream/pull/2085/commits/e88e9d9c05b9b142d0006144d1c7f09e3ee20750.

Screenshot 2021-06-29 at 19 18 45

Screenshot 2021-06-29 at 19 26 37

**Review by rudolfs — APPROVED** [review]

**Pull request merged**